app/uk/gov/hmrc/partnershipidentificationfrontend/models/BusinessVerificationStatus.scala

```diff
@@ -18,26 +18,30 @@ package uk.gov.hmrc.partnershipidentificationfrontend.models
 
 import play.api.libs.json._
 
-sealed trait  BusinessVerificationStatus
+sealed trait BusinessVerificationStatus
 
 case object BusinessVerificationPass extends BusinessVerificationStatus
 
 case object BusinessVerificationFail extends BusinessVerificationStatus
 
-case object BusinessVerificationUnchallenged extends BusinessVerificationStatus
+case object BusinessVerificationNotEnoughInformationToChallenge extends BusinessVerificationStatus
+
+case object BusinessVerificationNotEnoughInformationToCallBV extends BusinessVerificationStatus
 
 object BusinessVerificationStatus {
   val BusinessVerificationPassKey = "PASS"
   val BusinessVerificationFailKey = "FAIL"
-  val BusinessVerificationUnchallengedKey = "UNCHALLENGED"
+  val BusinessVerificationNotEnoughInfoToChallengeKey = "NOT_ENOUGH_INFORMATION_TO_CHALLENGE"
```

Comment:
It seems that we put the suffix Key to values and json "keys" ... I was wondering if it could be cleared if we group them with a context to separate possible value from the key itself ... not sure to be honest
Happy to accept this version without problem.

Reply:
This is a good point, it's a bit confusing. I can't think of anything clearer to rename it right now though. So might leave it